rules-tests/Php80/Rector/FunctionLike/UnionTypesRector/Fixture/skip_static_param.php.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\FunctionLike\UnionTypesRector\Fixture;
+
+class SkipStaticParam
```

Comment:
By convention, the "Skip" fixtures cannot have any changed content. That's how we know if the file should be untouched or not.

In this case, it would be more useful to have only single static param and test no change here.
Then have another fixture including the removed 2 lines from doc block.

Reply:
What about now

Reply:
Nailed it, thank you :+1: